### DISCUSSION TITLE
I was using the API to obtain unit information and got an error that could not be hashed.

### DISCUSSION THREAD

**Discussion by AirJerryWhite** [WeblateOrg/weblate#10932] [Q&A] [answered]
![418fa8073fb5e3c60c86eeb9cce559ee](https://github.com/WeblateOrg/weblate/assets/55286554/a54a5250-d8c0-460b-9fc9-f8454d2ac421)

**Comment by AirJerryWhite** [comment]
There was an error about duplicate entries before, but it has been fixed. How can I fix this error。

**Comment by nijel** [comment]
Can you please share whole exception?

**Comment by AirJerryWhite** [comment]
I need to contact server owner for a complete error report.Can I share something else like po file?

**Reply by nijel** [reply]
The exception traceback should not contain anything sensitive.

Anyway, the server is most likely using an outdated Weblate version as OrderedDict usage was dropped in 2022.

If you want to make an useful report, please include info like:

1. Server version.
2. What exactly you did.
3. The exception traceback.

**Comment by AirJerryWhite** [comment]
[weblate.txt](https://github.com/WeblateOrg/weblate/files/14231346/weblate.txt)
This is exception traceback.
I guessed it might be a problem with the language file. I set up a new project and added the entries, and the api could be used normally.

**Reply by AirJerryWhite** [reply]
We used polib to generate and update language files.

**Reply by nijel** [reply, marked as answer]
Thanks, I've created https://github.com/WeblateOrg/weblate/issues/10978 to track this issue.
